xds/internal/xdsclient/transport/transport.go

```diff
@@ -277,8 +284,11 @@ func (t *Transport) recvAggregatedDiscoveryServiceResponse(stream adsStream) (re
 	if err != nil {
 		return nil, "", "", "", fmt.Errorf("failed to read ADS response: %v", err)
```

Comment:
I think this the log message here can be improved. I understand that we are only tweaking the verbosity/levels  of logs.. But hear me out. 
In recv() we use this err message and log as 

```
t.logger.Warningf("ADS stream closed: %v", err)
```

so in the end, the log message looks like 
```
ADS stream is closed with error: failed to read ADS response: rpc error: code = Unavailable desc = error reading from server: read tcp 127.0.0.1:65270->127.0.0.1:65269: read: connection reset by peer
```

1. I think we can get rid of the dup description about ADS stream.
2. (not a part of this diff, and also not sure if would be a breaking change since we have some TAP tests run on error message comparisons): 
```
rpc error: code = Unavailable desc = error reading from server: read tcp 127.0.0.1:65270->127.0.0.1:65269: read: connection reset by peer
```

should maybe look like 

```
rpc error: code = Unavailable; desc = "error reading from server: read tcp 127.0.0.1:65270->127.0.0.1:65269: read: connection reset by peer"
```
?? Your thoughts?

Reply:
> I think this the log message here can be improved.

Thanks. Done. I changed it to return the error as is from `recvAggregatedDiscoveryServiceResponse`.

Regarding you other comment about changing the error message, that has to be a change in the status package. See: https://github.com/grpc/grpc-go/blob/5353eaa44095b31752e57f89034137f31fc7f763/internal/status/status.go#L139

I have no objections in changing that, but we can do that as a separate PR.

> we have some TAP tests run on error message comparisons:

That's a little worrying. If we are doing a `strings.Contains()` of the actual error message, we should be fine. But if we are doing a direct string comparison on the complete error message, that would be a problem.